src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -24,6 +24,7 @@
       RuntimeGraph="$(RuntimeIdentifierGraphPath)"
       NoWarn="$(NoWarn)"
       RunApiCompat="$([MSBuild]::ValueOrDefault('$(RunApiCompat)', 'true'))"
+      EnableStrictMode="$([MSBuild]::ValueOrDefault('$(RunApiCompat)', 'false'))"
```

Comment:
Is the RunApiCompat on by default? I think strict mode should have its own flag and should be on by default when running between compile and lib assets.